### PR TITLE
Related to zonemaster-engine issue 578

### PR DESCRIPTION
**Pull request by vlevigneron**
- Add method name (test case) in Logger.
- Fixes zonemaster/zonemaster-engine#578

DO NOT MERGE NOW, do a test first to check if it is what you are waiting for

### CODE REVIEW

**Comment by matsduf** [comment]
@vlevigneron, does this PR depend on the code change by zonemaster/zonemaster-engine/pull/762?

File: lib/Zonemaster/CLI.pm [hunk 1]
```diff
@@ -337,7 +346,7 @@ sub run {
         }
 
         if ( $self->config ) {
-            say $fh_diag __x( "Loading configuration from {path}.", path => $self->config );
+            say $fh_diag __x( "Loading configuaration from {path}.", path => $self->config );
```
Comment (mattias-p) [inline lib/Zonemaster/CLI.pm:349]: `configuaration` -> `configuration`

File: lib/Zonemaster/CLI.pm [hunk 2]
```diff
@@ -397,20 +406,25 @@ sub run {
                     }
 
                     if ( $self->show_level ) {
-                        printf "%-9s ", translate_severity( $entry->level );
+                        printf "%-9s ", __( $entry->level );
```
Comment (mattias-p) [inline lib/Zonemaster/CLI.pm:409]: This breaks msgid extraction. `xgettext` depends on `__()` only ever being called with literal (format) strings.

File: lib/Zonemaster/CLI.pm [hunk 3]
```diff
@@ -112,6 +114,13 @@ has 'show_module' => (
     default       => 0,
 );
 
+has 'show_testcase' => (
+    is            => 'ro',
+    isa           => 'Bool',
+    documentation => __( 'Print the name of the test case (method) on entries.' ),
```
Comment (mattias-p) [inline lib/Zonemaster/CLI.pm:120]: I don't think we should refer to test cases as methods at the interface level. The interface documentation shouldn't leak implementation details such as how test cases are implemented in code.

**Review by mattias-p — CHANGES_REQUESTED** [review]

**Comment by matsduf** [comment]
@vlevigneron, can this be completed?

File: lib/Zonemaster/CLI.pm (hunk 1 above)
Comment (vlevigneron) [reply lib/Zonemaster/CLI.pm:349]: Fixed

File: lib/Zonemaster/CLI.pm (hunk 2 above)
Comment (vlevigneron) [reply lib/Zonemaster/CLI.pm:409]: Sorry, I missed the point. It has been fixed.

File: lib/Zonemaster/CLI.pm (hunk 3 above)
Comment (vlevigneron) [reply lib/Zonemaster/CLI.pm:120]: You are right, I modify that.

**Comment by vlevigneron** [comment]
@matsduf @mattias-p Please review again and check if output is what you are waiting for.

**Review by matsduf — APPROVED** [review]

**Comment by matsduf** [comment]
@mattias-p, can you review again?

**Review by mattias-p — APPROVED** [review]

**Pull request merged**